### DISCUSSION TITLE
where can I see what has changed between version 35 and 36? Cant find the changelog on npmjs.org or in the repo

### DISCUSSION THREAD

**Discussion by mukundkatpatal** [renovatebot/renovate#23537] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I am getting all repos in all the projects scanned suddenly.
Although there is just one repository I have mentioned in the config.js
repositories: ['My Project/My.repository']

It was working fine in version 35

But since I have npx renovate I think new breaking version of 36 I see in the logs where It is scanning all the projects and all the repos

### What you tried so far.

I have tried searching for the change log to see what should I change for version 36. Otherwise I wil stay at 35

In the Debug: Combined config, I get repositories: [true], suddenly.
Previously, I was getting, repositories: [My project/my.repository]

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
https://github.com/renovatebot/renovate/releases/tag/36.0.0

**Comment by mukundkatpatal** [comment]
`Thanks,` I am getting the repository not found error in 35.143.0
In version 35.110.0 , renovate is running fine.

I have not made any changes to the config, or the yml. I dont see the major version changed here, still it started failing. How can I troubleshoot this?

The config.json looks like below

```
module.exports = {
  platform: "azure",
  endpoint: "https://dev.azure.com/myorg/",
  token: process.env.TOKEN,
  cloneSubmodules: false,
  hostRules: [
   {
      hostType: 'npm',
      matchHost: 'pkgs.dev.azure.com/myorg/_packaging/Shared/npm/registry/',
      username: 'apikey',
      password: process.env.TOKEN,
   },
   {
      baseUrl: "https://myorg.pkgs.visualstudio.com/_packaging/Shared/npm/registry/",
      hostType: "npm",
      token: process.env.TOKEN,
    },
    {
      baseUrl: "https://pkgs.dev.azure.com/myorg/_packaging/Shared/npm/registry/",
      hostType: "npm",
      token: process.env.TOKEN,
    },
    {
      matchHost: "https://myorg.pkgs.visualstudio.com/123456789/_packaging/Angular-Shared/npm/registry/",
      hostType: "npm",
      token: process.env.TOKEN
    }
  ],
  repositories: ["Shared Services/myorg.flare"]
};`

**Reply by rarkins** [reply]
If you can find the exact version where it changed then we have better chance if finding what changed. Alternatively look through the full diff and look for changes to the azure platform specifically
